pyproject.toml

```diff
@@ -51,7 +114,6 @@ exclude = [
     "cf_units/_udunits2_parser/parser",
 ]
 line-length = 79
-target-version = "py310"
```

Comment:
pre-commit rejects this as the same info can apparently be picked up from:

https://github.com/SciTools/cf-units/blob/8258d87b57f9a80eaa64769d7b36570b44e1dd26/pyproject.toml#L58

Reply:
:+1: as stated [here](https://docs.astral.sh/ruff/settings/#target-version)